tests/UI/specs/UIIntegration_spec.js

```diff
@@ -333,6 +333,20 @@ describe("UIIntegrationTest", function () { // TODO: Rename to Piwik?
         }, done);
     });
 
+    // Do not allow API response to be displayed
+    it('should not allow to widgetize an API call', function (done) {
+        expect.screenshot('widgetize_apidisallowed').to.be.captureSelector('#content', function (page) {
```

Comment:
This test seems to not work properly see https://travis-ci.org/piwik/piwik/jobs/68105052#L1337 

> Cannot find element #content